NitroxServer/Log.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System;
+using System.Diagnostics;
+
+namespace NitroxServer
+{
+    public class LogServer
```

Comment:
Instead of creating a new LogServer class can we keep the old one and have a function to pass in an optional textbox?  That way we can keep the same class and don't need to maintain two of the same loggers.  It would also allow us to get rid of the static Instance in MainWindow.